modules/mosdepth/main.nf

```diff
@@ -10,13 +10,13 @@ process MOSDEPTH {
     input:
     tuple val(meta), path(bam), path(bai)
     path  bed
-    val   window_size
```

Comment:
Might be a good idea to keep this in. I added it in because according to the docs you can only provide one or the other:

```
-b --by <bed|window>       optional BED file or (integer) window-sizes.
```

The `if/else` logic that has been deleted allows us to sanity check this and use one over the other. Have you tried running the module when providing both because if we assume that `--window` will be provided via `ext.args` then it could break the module if both are provided.

Another solution is to check `ext.args` to see if `--by` has been provided and use one over the other.

Reply:
Ok, but which one gets precedence then?